src/main/java/pages/YourShoppingCartPage.java

```diff
@@ -0,0 +1,139 @@
+
+    @FindBy(xpath = "//*[@id='email']")
+    WebElement emailFieldSignIn;
+
+    @FindBy(xpath = "//*[@id='passwd']")
+    WebElement passwordFieldSignIn;
+
+    @FindBy(xpath = "//*[@id='SubmitLogin']")
+    WebElement signInButton;
+
+    @FindBy(xpath = "//*[@id='add_to_cart']/button/span")
+    WebElement addToCartButton;
+
+    @FindBy(xpath = "//*[@class='icon-plus']")
+    WebElement increaseTheQuantityProductsButton;
+
+    @FindBy(xpath = "//*[@id='cart_title']//self::*[contains(text(),'Shopping-cart summary')]")
+    WebElement myCartPageWithProductsLabel;
+
+    @FindBy(xpath = "//*[@id='center_column']//ancestor::*[contains(text(),'Your shopping cart is empty.')]")
+    WebElement shoppingCartIsEmptyText;
+
+    @FindBy(xpath = "//*[@class='ajax_cart_no_product']")
+    WebElement shoppingCartIsEmptyHeaderText;
+
+    @FindBy(xpath = "//*[@id='layer_cart']//self::*[contains(text(),'Proceed to checkout')]")
+    WebElement proceedToCheckoutButton;
+
+    @FindBy(xpath = "//*[@id='center_column']//self::*[contains(text(),'Proceed to checkout')]")
+    WebElement proceedToCheckoutOnShoppingCartSummaryButton;
+
+    @FindBy(xpath = "//*[@id='cgv']")
+    WebElement iAgreeCheckBox;
+
+    @FindBy(xpath = "//*[@class='bankwire']")
+    WebElement payBankWire;
+
+    @FindBy(xpath = "//*[@id='cart_navigation']//ancestor::*[contains(text(),'I confirm my order')]")
+    WebElement paymentEndBtnIConf;
+
+    @FindBy(xpath = "//*[@class='cheque-indent']//ancestor::*[contains(text(),'Your order on My Store is complete.')]")
+    WebElement orderIsCompleteText;
+
+    @FindBy(xpath = "//*[@id='center_column']//ancestor::*[contains(text(),'My account')]")
+    WebElement myAccountPageLabel;
+
+    String productNameOnPageText = "//*[@id='homefeatured']//self::*[contains(text(),'%s')]";
+
+    String trashButton = "//*[@class='cart_quantity_delete']";
+
+    String quantityProductsOnCartPage = "//*[@class='cart_quantity_input form-control grey']";
+
+    String totalPriceSomeItemsProduct = "//*[@class='cart_total']";
+
+    public YourShoppingCartPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void fillInEmailAndClickSignInBtn(String emailAddress, String password) {
+        emailFieldSignIn.sendKeys(emailAddress);
+        passwordFieldSignIn.sendKeys(password);
+        signInButton.click();
+    }
+
+    public void findProductNameOnPage(String productNameItem) {
+        driver.findElement(By.xpath(String.format(productNameOnPageText, productNameItem))).click();
+    }
+
+    public void increaseProductsQuantity() {
+        increaseTheQuantityProductsButton.click();
+        addToCartButton.click();
+    }
+
+    public void checkedTabShipping() {
+        iAgreeCheckBox.click();
+        proceedToCheckoutOnShoppingCartSummaryButton.click();
+    }
+
+    public void clickProceedToCheckoutButton() {
+        proceedToCheckoutButton.click();
+    }
+
+    public void goToNextPage() {
```

Comment:
clickProceedToCheckoutButton

Reply:
fixed on goToNextPageProceedToCheckoutButton
because if we use clickProceedToCheckoutButton the driver don't finding the button with this locator